stacker/src/expull.rs

```diff
@@ -108,59 +84,68 @@ impl<'a> ExpUnrolledLinkedListWriter<'a> {
 
     #[inline]
     pub fn extend_from_slice(&mut self, mut buf: &[u8]) {
+        debug_assert!(buf.len() <= 32); // Check memcpy assumptions.
+        debug_assert!(!buf.is_empty()); // Check memcpy assumptions.
```

Comment:
```suggestion
```

Reply:
`extend_from_slice` is called only for byte slices of 1-18 (?), and the memcpy could be optimized for that, which I had tested.
I removed it, since I didn't want to add too many implicit assumptions.
The debug_asserts are outdated

Reply:
Did you change something? I do not see any change yet.

Reply:
I updated it now

Reply:
I still dont see any change.

Reply:
They are removed https://github.com/quickwit-oss/tantivy/pull/2058/commits/82ce3245e1adc92f5163cc55a4d5aa610ee26923